core/src/main/java/org/elasticsearch/monitor/jvm/JvmInfo.java

```diff
@@ -101,6 +108,14 @@
             info.memoryPools[i] = memoryPoolMXBean.getName();
         }
 
+        try {
+            HotSpotDiagnosticMXBean hotSpotDiagnosticMXBean = ManagementFactory.getPlatformMXBean(HotSpotDiagnosticMXBean.class);
+            VMOption usingCompressedOopsVmOption = hotSpotDiagnosticMXBean.getVMOption("UseCompressedOops");
+            info.usingCompressedOops = Boolean.parseBoolean(usingCompressedOopsVmOption.getValue());
+        } catch (Throwable t) {
+            info.usingCompressedOops = false;
```

Comment:
I think I'd prefer if it were just a string/enum/whatever that made the logs and api spit out "true"/"false"/"unknown".